apps/portals/src/configurations/elportal/synapseConfigs/studies.ts

```diff
@@ -27,9 +24,15 @@ export const studyCardConfiguration: CardConfiguration = {
   labelLinkConfig: [
     {
       isMarkdown: false,
-      baseURL: 'Explore/Projects/DetailsPage',
+      baseURL: 'Explore/Projects',
       URLColumnName: 'grant',
-      matchColumnName: 'grant',
+      matchColumnName: 'grants',
+    },
```

Comment:
One grant may have many projects, so link to the project explore page instead of project details page